### ISSUE TITLE
Recognize "Black Sats" in ord

### ISSUE THREAD

**Issue by RobertClarke** [ordinals/ord#2815]
"Black Sats" have risen in notoriety as interesting collectible sats due to their similarities with Rodarmor rarity Uncommons, Rares, etc., and having visual appeal ending in "...999999999"

Based on definitions on [Magisat](https://magisat.io/category/black-uncommon) and [Sating](https://docs.sating.io/sating.io-basics/the-main-sating-page/sating-search-types/black-sats), there seems to be consensus on the following:

- Black Uncommon Sats are the last sats in each new block.
- A Black Rare Sat is the last sat in each Bitcoin Difficulty Adjustment period. It is the sat just before a Rare.
- A Black Epic Sat is the last satoshi created in the coinbase of the last block before a Halving Event.

We could also enable the more rare ones (suggested):

- A Black Legendary sat could be the last sat in a cycle.
- A "Black Mythic" sat could be the last sat mined.

**Comment by casey** [comment]
I black sats, so I like the idea of recognizing them. Some thoughts:

- Would this be considered dilutive of the "original" rarity tiers?
- I want to come up with new names for each tier of black sat. I think it's confusing if there are two types of uncommons. I was thinking something like "curious, odd, anomalous, aberrant, bizarre" or "ink, pitch, jet, onyx, obsidian".
- What charms should we use? I was looking through emoji, and nothing immediately stood out for me.

**Comment by Magisat** [comment]
Love the idea of black sats being recognized <3. 

Q1: Would this be considered dilutive of the "original" rarity tiers?
No. People are already collecting them and are aware of their value proposition. Legitimizing them would be a good thing. 

Q2: I want to come up with new names for each tier of black sat. I think it's confusing if there are two types of uncommons. I was thinking something like "curious, odd, anomalous, aberrant, bizarre" or "ink, pitch, jet, onyx, obsidian".

A2: People are collecting and seeing value in them. Us and magic eden use the naming:
"uncommon black sat": last sat of the block
"rare black sat": last sat before difficulty adjustment
"epic black sat": last sat before halving
"mythical black sat": last sat created

I think it would be good to stick to these namings. The market and rare sat collectors are already used to identifying them by these names. 

Q3: What charms should we use? I was looking through emoji, and nothing immediately stood out for me.

A3: Maybe find some other seedling for the uncommon black sats 😂.

**Comment by RobertClarke** [comment]
> I black sats, so I like the idea of recognizing them. Some thoughts:
> 
> * Would this be considered dilutive of the "original" rarity tiers?
> * I want to come up with new names for each tier of black sat. I think it's confusing if there are two types of uncommons. I was thinking something like "curious, odd, anomalous, aberrant, bizarre" or "ink, pitch, jet, onyx, obsidian".
> * What charms should we use? I was looking through emoji, and nothing immediately stood out for me.

1. I agree with @Magisat's point on not being dilutive since all major marketplaces support both, and collectors find them worth collecting for their separately exciting qualities. Also, what's cooler than recognizing that the last sat ever mined will be a black/obsidian sat? 
2. I'm not strongly opinionated on the naming, and I like your suggestions! I think @Magisat has a fair point about the prevalent existing names. However, black sats are still not recognized for inscriptions on most significant marketplaces, so the meta hasn't proliferated far enough that we couldn't intentionally align on the names.
3. @cbspears's suggestions are good!

**Comment by cbspears** [comment]
> Black sats have really found their place in the market.
> 
> I don't believe they are dilutive. As @cbspears mentioned, the RRI (Rodarmor Rarity Index) is KING, market respects it.
> 
> I think the current naming works—black uncommon, black rare, black epic, etc. BUT not opposed to something else that makes sense.
> 
> Other emojis: ⚫◆❖🔲♣️♠️
> 
> *Deezy's sathunter is currently using the "black spade" for black uncommons fwiw

To note is that the Spade emoji doesn't display the same on different devices, explorers. That's one of the challenges here is having consistent emoji display everywhere.

**Comment by leonidasord** [comment]
In general I like the idea of Ordinal.com supporting more Satributes/charms, etc. What I really don't like though is that I don't know of a way to do that without it being viewed as Casey anointing additional rarities by indexing them in the Ordinals protocol.

I view the people chiming in on this discussion as the stewards of the emerging asset class of "rare sats" which has the potential to store trillions of dollars of value decades from now.

Even if that scenario doesn't seem likely I think that we owe it to those potential people in the future to act as if it may happen. In my opinion the creator of the "rare sats" asset class should not set the precedent of tampering with the rarities that were defined upon the release of the Ordinals protocol just as the creator of Bitcoin would not tamper with the supply of BTC after Bitcoin was released.

In summary I don't think it would be wise to have Casey in any way bring more rarities "into the fold" because it would break down what was previously a clear line of Casey's rarity and the community's rarity which would in my opinion would result in weird, hard to measure affects of the psychology of rare sats being a strong store of value.

The only reasonable argument around centralization of the rare sats asset class is Casey so this is one of those cases where I just want him to do nothing and let the market do it's thing to prevent a Vitalik type narrative from emerging.

**Comment by casey** [comment]
@leonidasord I'm definitely sympathetic to this argument. One thing that might help: These would be charms, not rarities. And not using the uncommon/rare/epic terminology might help make the distinction clearer.

I definitely think that I / the main implementation shouldn't support everything under the sun. Especially things, like palindromes, and all the exotic sat ranges, which have enormous supply.

But I think there has to be some path for adding new charms, since not adding anything ever seems like an unfortunate restriction.

I'm also fine just waiting on this for a bit, and seeing how things develop. My personal hunch is that the black sats will wind up being valued, because the supply is very scarce, and they feel like a mostly natural extension of the original rarity tier. So we could just wait for a few months or a year, and see if they've picked up enough steam to merit charms.

**Comment by BlackxBTC** [comment]
Hello Casey! Firstly, I want to express my gratitude for initiating this discussion. I am extremely excited about the potential integration of Black Sats into the Ordinals protocol, which would mark a significant full-circle moment. I would like to take this opportunity to personally address your questions and extend my thanks for starting this long-awaited conversation! 

**Question 1: Would this be considered dilutive of the "original" rarity tiers?**

On a personal note, I have consistently held the belief that Black Sats represent the culmination of the Rare Sat Thesis, always classifying them under the Rare Sats category. I base this classification primarily on their similar supply characteristics when compared to Uncommon Sats. Since its inception, I have maintained that Black Sats are not exotic. Every beginning must have an end, just as each block on Bitcoin marks both a commencement and a conclusion.

I firmly believe that the inclusion of Black Sats into the Ordinals protocol would not be dilutive, as they possess their own distinct markets. Over the past few months, Uncommon Sats have consistently demonstrated their rarity, with the floor price on all major rare sat marketplaces reflecting this. I argue that incorporating Black Sats will simply establish a new market rarity without impacting the existing tiers. This perspective has found support in the Magic Eden marketplace, where Black Sats have seen some of the highest volumes.

In fact, I am personally convinced that this integration will enhance the value of Uncommon Sats, elevating their worth. Black Sats, by following in the provenance of Uncommon Sats, will create a focal point for both Uncommon and Black Sats, contributing to their collective value.

**Question 2: I want to come up with new names for each tier of black sat. I think it's confusing if there are two types of uncommons. I was thinking something like "curious, odd, anomalous, aberrant, bizarre" or "ink, pitch, jet, onyx, obsidian".**

This is a topic that both White (Co-Founder of Classified) and I debated relentlessly when conceptualizing the idea of Black Sats. We initially considered naming tiers but ultimately discarded the idea due to concerns about provenance issues and potential confusion. In the end, we opted to retain the current naming convention for sats to pay homage to the original Sat Rarity thesis. We wanted to avoid introducing too many changes that could deviate significantly from the established tier ranking.

The market comprehends the current outline of rarity tiering, and I believe that straying from it could lead to considerable confusion (this is a personal belief). I am of the opinion that, given the similarity in Sat Rarity characteristics to Uncommon Sats, where Uncommon Sats are the first and Black Sats the last, it only makes sense to continue with the conventional naming system that was initially introduced.

**Question 3: What charms should we use? I was looking through emoji, and nothing immediately stood out for me.**

There is only one correct answer, and it has to be the ⬛ emoji. This is something I vehemently will stand by and hope everyone else feels the same way :)

**Conclusion:**
Thank you so much for initiating this discussion, and I look forward to your response! Building the Black Sat rarity over the last few months has been an utterly exhausting process as I have been simulatenously co-managing medical school. It means everything to finally see the Big Dogs paying some attention to it and I can't wait to find out what we all choose to proceed foward with. 

- Yours Truly, **𝙱 𝙻 𝙰 𝙲 𝙺**

**Comment by leonidasord** [comment]
> @leonidasord I'm definitely sympathetic to this argument. One thing that might help: These would be charms, not rarities. And not using the uncommon/rare/epic terminology might help make the distinction clearer.
> 
> I definitely think that I / the main implementation shouldn't support everything under the sun. Especially things, like palindromes, and all the exotic sat ranges, which have enormous supply.
> 
> But I think there has to be some path for adding new charms, since not adding anything ever seems like an unfortunate restriction.
> 
> I'm also fine just waiting on this for a bit, and seeing how things develop. My personal hunch is that the black sats will wind up being valued, because the supply is very scarce, and they feel like a mostly natural extension of the original rarity tier. So we could just wait for a few months or a year, and see if they've picked up enough steam to merit charms.

Black sats have clearly been determined as very valuable and the market has spoken on that. My point really has nothing to do with Black sats and more that rarities should not be messed with at the protocol level in any way once they were set in January. I think it would be cool to put Black Sats on Ordinals.com, but I don't want more sat rarities being indexed in the actual protocol itself for the store of value narrative if that makes sense. I just don't see a way to achieve that with the current setup.

The Open Ordinals Institute should not be in a position where people are coming to them asking for additional rarities to be added to the protocol. It's a recipe for trouble, especially as this asset class grows. That is an unnecessary point of centralization and it will have impacts on the market, and no one would rationally argue otherwise. It is not the role of the Open Ordinals Institute to issue/anoint tokens.

Also FWIW once a market picks a name it usually sticks and Black Uncommons, Black Rares, etc. are good names IMO that won't change easily.

**Comment by casey** [comment]
To be clear, The Open Ordinals institute doesn't have a say in this. Raph's work is funded by the OOI, but mine is not. The OOI is not a decision-making body, and does not guide what Raph and I work on.

**Comment by Magisat** [comment]
@leonidasord ord.io has the verification for accounts that own inscriptions. It could, in theory, establish a voting mechanism for such additions. Yes, I agree that @casey having the final say in such matters isn't the desirable way to move forward, nor is permanent stagnation (in terms of certain features, rarity rankings, charms, etc...). 

I would love to see ord.io, or even ordinals.com establish a voting mechanism with wallets holding at least one inscription (or running an ord indexer, or whatever criteria you see fit) being the eligible actors. So why don't we do that, and avoid public, unorganized debates?

**Comment by casey** [comment]
Democracy is not a particularly good way to make decisions. I don't think anyone can look at existing democratic processes and think, "Wow, that's a great decision-making mechanism that we should apply more widely."

**Comment by Magisat** [comment]
voting system that requires running an indexer, then? I doubt the uninformed average Joe runs an indexer.

**Comment by leonidasord** [comment]
I like the spirit of the idea but agree with Casey it would be a nightmare. The closest version of the idea that I could possibly see making sense would be to build some sort of abstract community rarity mechanism into the protocol and and reference explorer that wouldn't ever specify a specific rarity, rather it would allow anyone to "bring your own rarity"by letting you add the rules that you want (a math equation or a set of sats/ranges) that would allow the node operator to choose to index whatever rarities that they want.

**Comment by devords** [comment]
> My point really has nothing to do with Black sats and more that rarities should not be messed with at the protocol level in any way once they were set in January.

@leonidasord If these are added as charms, how would that be different from other charms that were recently added? Not disagreeing with you, just trying to understand the distinction.

**Comment by leonidasord** [comment]
> > My point really has nothing to do with Black sats and more that rarities should not be messed with at the protocol level in any way once they were set in January.
> 
> @leonidasord If these are added as charms, how would that be different from other charms that were recently added? Not disagreeing with you, just trying to understand the distinction.

I equally dislike all charms related to community sat rarity. They don't belong at the protocol level in my opinion. I like the charms related to the inscription part alone a lot though. It comes down to me not liking that the rare/exotic sat market now has an element to it of trying to convince Casey to anoint rarities by adding them to Ordinals at a protocol level and speculating on if it will happen or not. I've said many times that I think of these meta-protocols very similarly to blockchains and just as it would not be appropriate for people in the Ethereum ecosystem who hold a certain ERC-20 token to be campaigning Vitalik to add said token into Ethereum at the protocol level I don't wouldn't want the same dynamic here. I don't like the centralized aspect of it. One person having the power to add and remove tokens from the protocol is not a healthy dynamic to having in the market. All of these community/exotic rarities have a very healthy decentralized market right now and I don't want the creator of the protocol weighing in on which of those should be valued or not let alone adding them to the protocol. I realize I'm being the lame person in this convo, but I really have to express that I feel this dynamic that is emerging of trying to have the inside scoop on when the sat endpoint gets launched in order to be the first collection on it before a tweet goes out to the public, or trying to effectively get a token added to the protocol that you hold is not healthy. It is only human nature to do this. There is nothing wrong with people advocating for things that are in their financial interest. It is human nature to do so but it is up to the protocol creators to not let this dynamic emerge. I don't think it's healthy for the market to have this element to it.

**Comment by devords** [comment]
@leonidasord Thanks for explaining. I don't think it's lame at all. I more or less agree. I was surprised to see block 9 charm get added. All other charms are either Rodarmor sat rarity or related to a specific technical detail. So, the block 9 one stood out there as an odd one. Maybe it's a good one to remove.

**Comment by casey** [comment]
I've been hyping block nine sats before it was cool: https://twitter.com/rodarmor/status/1569883279326875648

**Comment by devords** [comment]
@casey Fair enough!

**Comment by leonidasord** [comment]
Can confirm that Casey predicted "historical/old" sat collecting (and everything that has transpired with rare sats) to a T! It's just a matter of what belongs in the core protocol or not. To me it's yucky for all of the reasons I shared above to be adding new tokens/rarities to the core protocol once a $100M+ market has already formed around the assets. It is not the protocol or it's creators role at this stage of things to be tampering with this stuff. It's really time to start treating changes to the protocol similar to how they would be treated on a blockchain storing billions of dollars of value. In my opinion the protocol really doesn't need to change a whole lot more after the Jubilee other than building out a robust set of recursive endpoints. Stability specifically coming from the founder is going to become increasingly important as the protocol grows. I get actual anxiety reading the GitHub because I don't see a reverence for stability and the billions of dollars of value being stored. This is just my honest thoughts. If Block 9s are indexed IMO so should Blacks (using Uncommon/Rare language) and probably a few others but I don't like any of it. I will leave it to yall as I have made my points about needing to be a lot more conservative and not messing with individual token markets at the protocol level which should be non controversial.

**Comment by BlackxBTC** [comment]
As we carefully consider the trajectory of our protocol, the prospect of seamlessly integrating with and nurturing the ordinals community through the Jubilee initiative is a compelling avenue. Our protocol's adaptability to mold and cultivate a promising future for this community is a critical aspect of its value proposition. However, as we navigate the landscape of market expansion and sustained growth, it is crucial to pragmatically evaluate the untapped potential within the billion-dollar Sat Rarities market. These conversations are serious and deserve a serious conversation that should happen over a live discourse as opposed to text. The implications go beyond GitHub.

Sat Rarities, being an undeniable historical artifact, occupies a distinctive position in the market. The unalterable origins, especially as the "OG" Sat Rarities (those first released upon the protocol release), make it an irreplaceable element of our narrative. Recognizing and embracing this historical significance is not just a strategic move; it is a fundamental consideration in its pursuit of market dominance and innovation on Bitcoin. Any attempt to overwrite this rich history with an arbitrary narrative risks devaluing the intrinsic worth that Sat Rarities carry. Black Sats does not infringe upon that history, and correlates with it. There is no difference between incorporating it on the protocol level or not, because irrespective, people are buying the asset. Collectors are collecting. The protocol should be open to support distinct movements that educates, onboards, and entertains people while introducing them to Ordinals. Black Sats provides that alternative for Bitcoiners as we speak. No one can take that away from the community.

In aligning our strategy with the acknowledgment of the profound implications inherent in the Bitcoin Blockchain, we position ourselves to pragmatically capitalize on the truth that markets and blockchains operate transparently. Instead of concealing this reality, our approach should be proactive, leveraging the authenticity and transparency that blockchain technology inherently provides with the market activity of Black Sats.

Amidst this backdrop, the emergence of Black Sats and their associated rarity represents a formidable force that cannot be dismissed lightly. The unique qualities and scarcity of Black Sats hold significance within the rarest segments of the market, the uncommon category. Neglecting or sidelining this emerging trend would equate to overlooking a valuable opportunity to tap into a distinct segment of the market and expand the ordinals community.

This strategic shift towards recognizing and integrating the historical value of Sat Rarities, especially Black Sats, transcends being a mere business decision; it embodies a commitment to the Ordinals community. Without their active participation and engagement, the realization of the envisioned billion dollar market is hard to imagine. By acknowledging the depth of history, the transparency of the blockchain, and the weight of emerging trends, we position ourselves not just as innovators but as responsible stewards of a community-driven narrative that goes beyond conservative market dynamics.

**Comment by casey** [comment]
I'm not sold on adding black sats, but I think that adding charms for sat categories that have very even distribution is completely different from endorsing a particular token. If there are bag holders of a sat category with very even distribution, the vast majority of the supply will still be unsequestered.

**Comment by leonidasord** [comment]
> I'm not sold on adding black sats, but I think that adding charms for sat categories that have very even distribution is completely different from endorsing a particular token. If there are bag holders of a sat category with very even distribution, the vast majority of the supply will still be unsequestered.

My final point then I will shut up I promise!

Black sats are as legit in the market as any other exotic sat right now and have very good distribution. This has nothing to do with Black sats and everything to do with effectively the Vitalik figure of Ordinals crossing a line into talking about what tokens are valuable and more so which should be codified into the protocol. Vitalik saying that xyz token has good distribution and therefor we can code it into Ethereum would be pretty wild. There is a beauty in keeping the core protocol agnostic and allowing the community to build on top without needing changes to the protocol itself. 

Black Uncommons have an MCAP of ~$65M right now and Uncommons have an MCAP of ~$167M. That is a real market with real market participants spending millions of hard earned dollars and it is simply not appropriate for the founder of the protocol to be weighing in on that ratio or choosing to retain the power of anointing or not anointing any tokens for any reason. A very slight change in that ratio is millions of dollars in value being creating/destroyed due to the creator of the protocol injecting themself into the market. That is simply not appropriate and an unhealthy dynamic in the market. The creator of the protocol's role is to focus on stewarding the ecosystem, not weighing in on the price dynamics like the rest of us have the luxury of being able to do. It's a large responsibility and a difficult position to be in. Again just my honest thoughts. I want to see this ecosystem thrive including its founder and that means having a conservative, principled approach to protocol development (in my opinion).

**Comment by BlackxBTC** [comment]
I actually think it is the opposite to be honest Leonidas. Casey has admitted to being hesitant to add Black Sats since the beginning and I think the community is dictating this one, more than a singular founder. I think it is more of recognizing what the Ordinals community represents as being worthy of being added to the protocol rather than one sole founder, and in this case, we are simply talking about charms, which allows inscriptions to be tagged directly on chain using the original protocol. 

Furthermore, I do not necessarily enjoy comparing Casey to Vitalik because I believe that creators should be allowed to create. This is not Ethereum, and does not compare to the expectations that Ethereum has. We actively expanding with the value of on-chain rarities still being discovered. I do not really see the comparison being made here. Bitcoin Ordinals and Rare Sats are in their own respective category and should be handled as such without fear of repeating the past. History repeats itself only when these careful conversations do not occur. The culture on Bitcoin is different and this is not comparable to adding random shitcoins into the protocol or the "creator injecting himself into the market".

Holding the Ordinals Protocol and its founders hostage for even simple additions such as charms, which are inconsequential in terms of "on-chain value" does not really make sense. I understand the need to be conservative, but I am not afraid to push the fold for something I think can benefit the entire Ordinals community, not only in terms of financial value, but community value as well. When deciding implementations for the future, we should not alter the past, but that also does not mean that we do not embrace the future. A simple charm to recognize what sat rarity you are viewing does not really take away or diminish the value of anything. In fact, it ADDS value.

I believe that a Black Sat charm, Block 9 Charm, Block 78 charm, and Palindrome charm, are things that should be added because it is what the Ordinals and Bitcoin community believes in. Adding these charms to the protocol and officially recognizing them does not superimpose anything other than what is reality. Collectors are collecting and want to know what they are collecting. The reality is that these Sat Rarities will most likely outgrow the protocol to begin with. I strongly believe that recognizing assets that are being denominated and sold for Bitcoin, should be included into the Ordinals protocol. It makes logical sense. These are not random shitcoins that are being pumped out left and right. As you stated Leonidas, we are talking about a good chunk of Bitcoin here in terms of market share... ignoring market participants for a market evaluation of Black Sats that has roughly half the value of Uncommon Sats can create devestating and unintended consequences, especially if changes are made to include other Sat Rarity markets....

There is inherent scarcity and we should welcome all market participants with open arms and inclusion, especially for something simple like Charms. We are talking about Bitcoin Sat Rarities here, there is no comparison to other Blockchains. Especially not a centralized one. If there was an addition made for one Sat Rarity due to high demand and market evaluation, it should be made for all because the Bitcoin community supports that creation of value and without it, we are nothing.

I will provide an analogy with the Mona Lisa to demonstrate my point:

The reason the Mona Lisa is worth roughly 20,000 BTC is not because Leonardo Da Vinci stated its worth. It occurred because in August 1911 it was stolen by a former employee of the Louvre who, convinced that the painting belonged to Italy and should not remain in France, decided to steal it.

Newspapers worldwide picked up the story of the art heist and the ensuing search and ran with it for years, making the Mona Lisa the most famous painting of its day - and probably of all time.

Which raises another question:

Why might it be so valuable to someone to own such a famous painting?

Simple. Because owning the Mona Lisa would add your name to the story of the Mona Lisa.

It’s a story that started before we were born and will probably continue after we are gone.

To a select few, $850,000,000 would be a small price to pay to be remembered forever.

Black Sats are no different. Owning a Black Sat adds to the story of Black Sats and Rare Sats on Bitcoin.

It's a story that started before I joined Ordinals and will continue after I die.

Here’s the thing...

When it comes to creating value on Bitcoin, intangibles matter. The story and commmunity building that story on Bitcoin matters. Black Sats and all other Sat Rarities on Bitcoin matter. Scarcity on Bitcoin matters. Representation and demonstration of market value on Bitcoin matters.

Those are my 2 Sats. Thoroughly enjoying the conversation and excited to see what unfolds!

**Comment by cbspears** [comment]
> The Open Ordinals Institute should not be in a position where people are coming to them asking for additional rarities to be added to the protocol. It's a recipe for trouble, especially as this asset class grows. That is an unnecessary point of centralization and it will have impacts on the market, and no one would rationally argue otherwise.

I would argue that this is adding a sat rarity recognizance to the reference client which _implements_ the protocol. However it seems that no matter how hard I try to emphasize this and encourage the market to distinguish between the protocol and the client implementation, the market insists on conflating the two.

This is just my opinion and I refer to my repeated call for better protocol definitions & documentation, or at least better communication on this topic. We see this same issue cause dispute by the market believing that the ordinals.com front page explorer is also the protocol itself and not a representation of a "lens" to view the protocol through, e.g. the JSON display debate.

I expect failure to distinguish between protocol <> implementation <> explorer will only continue to create perceived arguments & disputes over governance and consensus when most of these could be avoided.

Apologies for possibly derailing the conversation, if we want to keep discussing this it might be best to spin up another Discussion. I just thought it was a relevant place to chime in on the topic of what I believe is the protocol and what isn't.

**Comment by casey** [comment]
I think I'm going to close this. We can revisit it later. My reasons are:

- It's confusing to add charms/rarities which have the same name (uncommon/rare/epic/legendary) as the existing rarity tier.
- I can't think of good emoji / charms.
- It does feel a little dilutive to me of the existing rarity tiers.
